backend/remote-state/manta/backend.go

```diff
@@ -131,6 +131,9 @@ func (b *Backend) configure(ctx context.Context) error {
 	if data.Get("key_id").(string) == "" {
 		validationError = multierror.Append(validationError, errors.New("`Key ID` must be configured for the Triton provider"))
 	}
+	if data.Get("key_id").(string) == "" {
+		validationError = multierror.Append(validationError, errors.New("`Key ID` must be configured for the Triton provider"))
```

Comment:
This seems the same as the block above it ... :thinking: is there a subtle difference I'm not seeing?

Reply:
Yes! The difference is "I need to update that with the correct attribute"  🤣 thank you so much

Reply:
On second thought, that shouldn't be in there at all - removed!